readme.md

```diff
@@ -184,7 +197,7 @@ Then, go to [http://localhost:5000/](http://localhost:5000) and you should see a
 * **Question:** The above can be described as middleware. Describe what middleware is in your own words and how it differs from building a route.
 
 * **Your Answer:**
-
+middleware processes or validates the request in some way.  This can be used regardless of the route to process an incoming request
```

Comment:
Middleware is the code/application/module that communicates between two systems, such as between servers and databases. The code above is different than a route because it is always run whenever the application gets to these lines.